pegasus/sites.v3/code.org/views/carousel_inspirational_video.haml

```diff
@@ -2,7 +2,7 @@
 
 - inspirational_video_caption = [hoc_s(:video_library_page_inspirational_video_title_9), hoc_s(:video_library_page_inspirational_video_title_8), hoc_s(:video_library_page_inspirational_video_title_3), hoc_s(:video_library_page_inspirational_video_title_15), hoc_s(:video_library_page_inspirational_video_title_17), hoc_s(:video_library_page_inspirational_video_title_1), hoc_s(:video_library_page_inspirational_video_title_7), hoc_s(:video_library_page_inspirational_video_title_2), hoc_s(:video_library_page_inspirational_video_title_4), hoc_s(:video_library_page_inspirational_video_title_10), hoc_s(:video_library_page_inspirational_video_title_5), hoc_s(:video_library_page_inspirational_video_title_11), hoc_s(:video_library_page_inspirational_video_title_6), hoc_s(:video_library_page_inspirational_video_title_12), hoc_s(:video_library_page_inspirational_video_title_13), hoc_s(:video_library_page_inspirational_video_title_14), hoc_s(:video_library_page_inspirational_video_title_16)]
 
-- video_inspirational_download = ["", "//videos.code.org/social/cs-is-everything.mp4", "//videos.code.org/social/ChangeTheWorld_v5.mp4", "", "//videos.code.org/social/Pride_Month_2024.mp4", "//videos.code.org/social/change_the_face_of_cs.mp4", "//videos.code.org/social/athlete-steph-curry.mp4", "//videos.code.org/social/athletes-hour-of-code.mp4", "//videos.code.org/social/what-most-schools-dont-teach.mp4", "//videos.code.org/2015/social/president_obama_asks_america.mp4", "", "//videos.code.org/social/codeorg-60sec.mp4", "//videos.code.org/social/my-name-is-aracely.mp4", "//videos.code.org/social/latam-hour-of-code-2018.mp4", "", "", "//videos.code.org/ywics-panel-2023.mp4"]
+- video_inspirational_download = ["", "//videos.code.org/social/cs-is-everything.mp4", "//videos.code.org/social/ChangeTheWorld_v5.mp4", "", "//videos.code.org/social/Pride_Month_2024.mp4", "//videos.code.org/social/change_the_face_of_cs.mp4", "//videos.code.org/social/athlete-steph-curry.mp4", "//videos.code.org/social/athletes-hour-of-code.mp4", "//videos.code.org/social/what-most-schools-dont-teach.mp4", "//videos.code.org/2015/social/president_obama_asks_america.mp4", "//videos.code.org/HoC-video-15mb.mp4", "//videos.code.org/social/codeorg-60sec.mp4", "//videos.code.org/social/my-name-is-aracely.mp4", "//videos.code.org/social/latam-hour-of-code-2018.mp4", "", "", "//videos.code.org/ywics-panel-2023.mp4"]
```

Comment:
Added fallback video here now that it's been moved to the `videos.code.org` S3 bucket.